src/components/user/user-detail-editor/UserDetailEditor.tsx

```diff
@@ -0,0 +1,559 @@
+                </Grid>
+                <Grid item xs={12}>
+                  <TextField
+                    data-testid="phone-field"
+                    InputProps={{
+                      startAdornment: (
+                        <InputAdornment position="start">+47</InputAdornment>
+                      ),
+                    }}
+                    inputProps={{
+                      inputMode: "numeric",
+                      pattern: "[0-9]{8}",
+                    }}
+                    required
+                    fullWidth
+                    id="phoneNumber"
+                    label="Telefonnummer"
+                    autoComplete="tel-national"
+                    error={!!errors.phoneNumber}
+                    {...register("phoneNumber", fieldValidators.phoneNumber)}
+                  />
+                  <FieldErrorAlert error={errors.phoneNumber} />
+                </Grid>
+                <Grid item xs={12}>
+                  <TextField
+                    data-testid="address-field"
+                    required
+                    fullWidth
+                    id="address"
+                    label="Adresse"
+                    autoComplete="street-address"
+                    error={!!errors.address}
+                    {...register("address", fieldValidators.address)}
+                  />
+                  <FieldErrorAlert error={errors.address} />
+                </Grid>
+                <Grid item xs={12}>
+                  <TextField
+                    data-testid="postal-code-field"
+                    InputProps={{
+                      endAdornment: (
+                        <InputAdornment position={"end"}>
+                          {postalCity}
+                        </InputAdornment>
+                      ),
+                    }}
+                    inputProps={{ inputMode: "numeric", pattern: "[0-9]{4}" }}
+                    required
+                    fullWidth
+                    id="postalCode"
+                    label="Postnummer"
+                    autoComplete="postal-code"
+                    error={!!errors.postalCode}
+                    {...register("postalCode", {
+                      // Need to have a separate onChange because of autofill not triggering validation
+                      onChange: onPostalCodeChange,
+                      ...fieldValidators.postalCode,
+                    })}
+                  />
+                  <FieldErrorAlert error={errors.postalCode} />
+                </Grid>
+                <Grid item xs={12}>
+                  <Controller
+                    control={control}
+                    {...register("birthday", fieldValidators.birthday)}
+                    name="birthday"
+                    render={() => {
+                      return (
+                        <DatePicker
+                          sx={{ width: "100%" }}
+                          label="Fødselsdato"
```

Comment:
This field is required, and since it uses a custom component I guess we'll have to add the asterisk manually.
```suggestion
                          label="Fødselsdato *"
```

Reply:
✅